common/src/main/java/dev/latvian/mods/kubejs/item/ItemBuilder.java

```diff
@@ -92,6 +92,9 @@ public static ArmorMaterial ofArmorMaterial(Object o) {
 	public transient FoodBuilder foodBuilder;
 	public transient Function<ItemStackJS, Color> barColor;
 	public transient ToIntFunction<ItemStackJS> barWidth;
+
+	public transient Function<ItemStackJS, Component> getName;
```

Comment:
A better name for this field would be `nameGetter`, as it only "gets" the name when dereferenced with the invocation method.